src/Webhooks/Webhook.cs

```diff
@@ -41,22 +47,23 @@ public Webhook(ulong id, string token) : this($"https://discordapp.com/api/webho
 
         public void Send()
         {
-            Core.Instance.SvManager.StartCoroutine(PostRequest());
+            SvManager.Instance.StartCoroutine(PostRequest());
```

Comment:
You still set Core.Instance.SvManager in the ManagerStart event so there's no reason to change this line

Reply:
Sorry for that, I was testing the code to find the error, and the SvManager from Core.Instance was null so I changed it to SvManager.Instance, I think it won't reduce the performance. But if you want i can change that line